changelog.d/10055.misc

```diff
@@ -0,0 +1 @@
+Remove redundant, unmaintained `convert_server_keys` script.
```

Comment:
Should this be a `.removal`?

Reply:
I figure since it was in scripts-dev, it's more a misc thing.